azure-pipelines.yml

```diff
@@ -342,6 +343,7 @@ jobs:
           if git tag v$(release_tag) $(release_sha); then
             git push origin v$(release_tag)
             mkdir $(Build.StagingDirectory)/release
+            mkdir $(Build.StagingDirectory)/beta-release
```

Comment:
I think the directory is more confusing than helpful given that it’s currently alpha, will be beta at some point and you will have to keep changing that. If you do want a directory maybe just call it `daml-on-sql`? or `fat-jars`?